src/main/java/org/sonarlint/intellij/ui/SonarLintAnalysisResultsPanel.java

```diff
@@ -0,0 +1,163 @@
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.ui;
+
+import com.intellij.ide.OccurenceNavigator;
+import com.intellij.ide.util.PropertiesComponent;
+import com.intellij.openapi.actionSystem.DataProvider;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.ComboBox;
+import com.intellij.openapi.vcs.ProjectLevelVcsManager;
+import com.intellij.ui.ScrollPaneFactory;
+import com.intellij.util.messages.MessageBusConnection;
+import com.intellij.util.ui.tree.TreeUtil;
+import java.awt.BorderLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.Insets;
+import javax.swing.Box;
+import javax.swing.DefaultComboBoxModel;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.sonarlint.intellij.core.ProjectBindingManager;
+import org.sonarlint.intellij.messages.StatusListener;
+import org.sonarlint.intellij.ui.scope.AbstractScope;
+import org.sonarlint.intellij.ui.scope.AllFilesScope;
+import org.sonarlint.intellij.ui.scope.ChangedFilesScope;
+
+public class SonarLintAnalysisResultsPanel extends AbstractIssuesPanel implements OccurenceNavigator, AbstractScope.ScopeListener, DataProvider {
+  private static final String SELECTED_SCOPE_KEY = "SONARLINT_ANALYSIS_RESULTS_SCOPE";
+  private static final String SPLIT_PROPORTION_PROPERTY = "SONARLINT_ANALYSIS_RESULTS_SPLIT_PROPORTION";
+
+  private final LastAnalysisPanel lastAnalysisPanel;
+  private AbstractScope scope;
+
+  private ChangedFilesScope changedFilesScope;
+  private ComboBox scopeComboBox;
+
+  public SonarLintAnalysisResultsPanel(Project project, ProjectBindingManager projectBindingManager) {
+    super(project, projectBindingManager);
+    this.lastAnalysisPanel = new LastAnalysisPanel(project);
+
+    // Issues panel with tree
+    JPanel issuesPanel = new JPanel(new BorderLayout());
+    issuesPanel.add(createScopePanel(), BorderLayout.NORTH);
+    issuesPanel.add(ScrollPaneFactory.createScrollPane(tree), BorderLayout.CENTER);
+    issuesPanel.add(lastAnalysisPanel.getPanel(), BorderLayout.SOUTH);
+    setToolbar(scope.toolbarId());
+
+    // Put everything together
+    super.setContent(createSplitter(issuesPanel, detailsTab, SPLIT_PROPORTION_PROPERTY, false, 0.65f));
+
+    // Subscribe to events
+    subscribeToEvents();
+  }
+
+  private void subscribeToEvents() {
+    MessageBusConnection busConnection = project.getMessageBus().connect(project);
+
+    //  busConnection.subscribe(ProjectLevelVcsManager.VCS_CONFIGURATION_CHANGED,
+    //       () -> ApplicationManager.getApplication().invokeLater(this::vcsChange));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)